packages/expo-image/src/Image.types.ts

```diff
@@ -108,6 +108,14 @@ export type ImageProps = AccessibilityProps & {
    */
   cachePolicy?: ImageCachePolicy | null;
 
+  /**
+   * Determines whether to choose image source based on container size only on mount or on every resize.
+   * Use `live` to improve performance.
+   * @default "initial"
```

Comment:
You've changed it to `live` in `useSourceSelection` 😅

Reply:
See, this is why I can't code and talk 🤦